.github/workflows/all-prod-tn-staging.yml

```
@@ -0,0 +1,99 @@
+  push:
+    branches:
+      - all-1.4.0.1
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: all-prod-rig
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)